app/src/util_ios.h

```diff
@@ -24,6 +24,7 @@
 
 #include <map>
 #include <string>
+#include <unordered_set>
 #include <vector>
 
 #include "app/src/include/firebase/variant.h"
```

Comment:
I think you can fix the lint error if you move the system #includes above the #ifdef __OBJC__.